pkgs/tools/admin/google-cloud-sdk/update.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+BASE_URL="https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk"
+
+# Version of Google Cloud SDK from
+# https://cloud.google.com/sdk/docs/release-notes
+VERSION="364.0.0"
```

Comment:
Can we auto fetch that number?

Reply:
I haven't found a nice way to do that apart from HTML scraping. So this script only updates the nix files with the respective sha's.

Reply:
we could do that or I found this https://github.com/twistedpair/google-cloud-sdk/blob/master/.github/workflows/sync_gcloud_releases.sh#L103

Reply:
thanks for the link. it looks like it requires an authentication token to check the GCS bucket so we would have to create a token. So I think it's better to manually update it until there's a proper way to get release versions automatically and easily.

Reply:
Yeah, I thought we can somehow easily get it but oh well...